cocos/spine/skeleton.ts

```diff
@@ -975,7 +975,6 @@ export class Skeleton extends UIRenderer {
             this._animationName = name;
             trackEntry = this._instance!.setAnimation(trackIndex, name, loop);
         }
-        this._markForUpdateRenderData();
```

Comment:
This logic does not exist at jsb-spine-skeleton.js(native platform). So remove it to synchronize with native.